src/uu/tail/src/tail.rs

```diff
@@ -169,16 +180,16 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         }
     };
 
-    if given_options.opt_present("z") {
+    if matches.is_present("z") {
         if let FilterMode::Lines(count, _) = settings.mode {
             settings.mode = FilterMode::Lines(count, 0);
         }
     }
 
-    let verbose = given_options.opt_present("v");
-    let quiet = given_options.opt_present("q") || given_options.opt_present("silent");
+    let verbose = matches.is_present("v");
+    let quiet = matches.is_present("q") || matches.is_present("silent");
 
-    let files = given_options.free;
+    let files = matches.is_present("free");
```

Comment:
to get files, please replicate this approach:
https://github.com/uutils/coreutils/blob/master/src/uu/chown/src/chown.rs#L152-L156
https://github.com/uutils/coreutils/blob/master/src/uu/chown/src/chown.rs#L164-L167